.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,30 @@
 ## Linked Issue
 
-_[Whenever possible, a pull request should address a pre-existing issue.
-That issue can be linked to the pull request by using the side panel in the Github UI or
-using the `#` symbol followed by the number of the associated issue]_
+If you need tips on filling out these sections, click the <> icon at the top of the file to disable Markdown rendering. 
```

Comment:
So, since this is a Pull Request template, the entire contents of this file get dumped in to the Pull Request description:

![image](https://user-images.githubusercontent.com/360261/136209735-b9aa0040-b749-4f5b-8290-ac39fe3efd62.png)

As you can see, we don't need to worry about hiding/showing comments. What we want is that text to be visible to people when they're creating a PR, but hidden once they hid save if they don't delete it.